Wikipedia/Code/WebPageUserScript.swift

```diff
@@ -1,7 +1,9 @@
 /// PageUserScript uses the platform-appropriate WKUserScript initializer to create a user script in WKContentWorld.page
 class PageUserScript: WKUserScript {
     override init(source: String, injectionTime: WKUserScriptInjectionTime, forMainFrameOnly: Bool) {
-        if #available(iOS 14.0, *) {
+        if #available(iOS 14.0, *), UI_USER_INTERFACE_IDIOM() == .phone {
+            // For some reason this crashes on iPadOS.
```

Comment:
@joewalsh I'm seeing the opposite actually in Beta 5. With this line iPad reliably crashes when it enters the `else`. Can you confirm this is fixed now & we can remove the `.phone` check?

Reply:
Confirmed, thanks for fixing this!